bcda/api_test.go

```diff
@@ -701,13 +701,14 @@ func (s *APITestSuite) TestAuthToken() {
 	handler.ServeHTTP(s.rr, req)
 
 	assert.Equal(s.T(), http.StatusBadRequest, s.rr.Code)
-	assert.Empty(s.T(), s.rr.Body)
 
-	req.Header.Add("client_id", "not_a_client_id")
-	req.Header.Add("secret", "not_a_secret")
+	// // The following change in error status can be tested with alpha auth (presumably along with success paths)
+	// // once GetAccessToken() is implemented
+	//
+	// req.Header.Add("client_id", "not_a_client_id")
+	// req.Header.Add("secret", "not_a_secret")
```

Comment:
FYI, this is supposed to be a basic authentication header, which is one header value, not two. See https://en.wikipedia.org/wiki/Basic_access_authentication.